packages/now-cli/test/integration.js

```diff
@@ -143,6 +143,15 @@ const apiFetch = (url, { headers, ...options } = {}) => {
   });
 };
 
+const getDeploymentBuildsByUrl = async url => {
+  const query = new URLSearchParams({ url });
+  const hostRes = await apiFetch(`/v10/now/deployments/unkown?${query}`);
```

Comment:
Is this meant to say `unknown`? 🤔

Reply:
@rdev yes, because we would need to pass a `deploymentId` here, but we don't have one, so we need to use something else.

Reply:
The reason is that we will use `url` instead of the `deploymentId`, but due to routing we can't just leave this segment of the URL.

Reply:
@andy no no, I mean it looks like there's a typo - `unkown` vs `unknown`

Reply:
haha, yea, you are right

Reply:
Hmm, the docs say to use `/v10/now/deployments/get?url=`


https://zeit.co/docs/api#endpoints/deployments/get-a-single-deployment

Reply:
@styfle sure, but you can write here whatever you want, since it gets ignored when `url` exists